### PR TITLE
[directx-headers] updated for DirectX SDK Agility SDK v1.602.0

### PR DESCRIPTION
**Pull request by walbourn**
Updates the *directx-headers* port with the latest release of the [DirectX Agility SDK](https://github.com/microsoft/DirectX-Headers/releases/tag/v1.602.0) headers.

https://devblogs.microsoft.com/directx/agility-sdk-1-602-0/

### CODE REVIEW

**Comment by walbourn** [comment]
Verified the following supported triplets:

```
vcpkg install directx-headers:x86-windows
vcpkg install directx-headers:x86-windows-static
vcpkg install directx-headers:x86-windows-static-md
vcpkg install directx-headers:x64-windows
vcpkg install directx-headers:x64-windows-static
vcpkg install directx-headers:x64-windows-static-md
vcpkg install directx-headers:arm64-windows
vcpkg install directx-headers:arm64-windows-static
vcpkg install directx-headers:arm64-windows-static-md
vcpkg install directx-headers:x86-uwp
vcpkg install directx-headers:x64-uwp
vcpkg install directx-headers:arm-uwp
vcpkg install directx-headers:arm64-uwp

./vcpkg install directx-headers:x64-linux

./vcpkg install directx-headers:arm64-linux
```

**Comment by walbourn** [comment]
@JonLiu1993 This PR is ready to go now.

File: versions/d-/directx-headers.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "f6271f7dc1af5888b39b0443ff7ecc9d01f1db94",
+      "version": "1.602.0",
+      "port-version": 0
+    },
+    {
+      "git-tree": "cac371748d8d7627e43890fd109d05e5ee48d105",
+      "version": "1.600.10",
+      "port-version": 0
+    },
```
Comment (BillyONeal) [inline versions/d-/directx-headers.json:12]:
```suggestion
```

Hmmm the bot should have caught this?

Comment (walbourn) [reply versions/d-/directx-headers.json]: Fixed.

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**